README.md

```diff
@@ -2,85 +2,388 @@
+
+Since Cockpit was created and is maintained by Blue Robotics, we activelly test the application everyday against ArduSub and ArduRover vehicles, which we sell, so you can expect the best experience with those two types of vehicles.
+
+We currently consider the application to fully cover Submarine missions, and partially cover Boat missions, since those are usually more complex, requiring dedicated map-based navigation commands, like loitering, geofencing and servo controlling, which we don't support yet. As of now, Cockpit only supports regular waypoints and survey polygon-based missions.
+
+ArduCopter and ArduPlane have received initial support on Cockpit, with dedicated widgets for Takeoff and Landing, with ArduCopter being done with real-life tests. It should be noticed thought that we don't perform regular tests on those, so use it at your own risk.
+
+<div align="center">
+  <table>
+    <tr>
+      <td align="center">
+        <img src="src/assets/brov2-marker.png" width="80"><br>
+        <strong>Submarines</strong><br>
+        <em>ArduSub</em>
+      </td>
+      <td align="center">
+        <img src="src/assets/blueboat-marker.png" width="80"><br>
+        <strong>Surface Boats</strong><br>
+        <em>ArduRover</em>
+      </td>
+      <td align="center">
+        <img src="src/assets/arducopter-top-view.png" width="80"><br>
+        <strong>Drones</strong><br>
+        <em>ArduCopter</em>
+      </td>
+    </tr>
+  </table>
+</div>
+
+---
+
+## 🎨 Features Overview
+
+### 🖥️ **Customizable Interface**
+- **Drag-and-drop widgets** for complete layout control
+- **Multiple views** for different operational modes
+- **Responsive design** that works on all screen sizes
+- **Dark/light themes** with customizable glass effects
+
+### 🎮 **Vehicle Control**
+- **ArduPilot support** for submarines, drones, planes, and rovers
+- **MAVLink protocol** for reliable communication
+- **Joystick integration** with support for Xbox, PlayStation, and other controllers
+- **Custom button mappings** for different vehicle types
+- **Real-time telemetry** with customizable data displays
+
+### 📹 **Video Streaming**
+- **WebRTC-based streaming** for low-latency video
+- **Multi-stream support** with stream mapping
+- **Video recording** with automatic processing and metadata
+- **Snapshot capture** with GPS coordinates and telemetry
+- **Video statistics** for monitoring stream quality
+
+### 🗺️ **Mission Planning**
+- **Interactive map interface** with drag-and-drop waypoints
+- **Survey patterns** for automated area coverage
+- **Points of interest** management
+- **Mission import/export** in standard formats
+- **Real-time mission monitoring** and execution
+
+### 📊 **Data Management**
+- **Data Lake** for centralized variable storage
+- **Telemetry logging** with customizable parameters
+- **Export capabilities** for data analysis
+- **Real-time plotting** and visualization
+- **Historical data playback**
```

Comment:
When was this added‽ Also definitely needs to be documented.

Reply:
The historical data playback was AI non-sense that I didn't catch 😝